### ISSUE TITLE
Conclusion: remove symfony/console v4.0.8

### ISSUE THREAD

**Issue by polishedwp** [weprovide/valet-plus#153]
Still having problems install. I tried the fixes for the closed issue but does not fix my problem. 

`composer global require weprovide/valet-plus
Changed current directory to /Users/webrev/.composer
Using version ^1.0 for weprovide/valet-plus
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install weprovide/valet-plus 1.0.13
    - Conclusion: don't install weprovide/valet-plus 1.0.12
    - Conclusion: don't install weprovide/valet-plus 1.0.11
    - Conclusion: don't install weprovide/valet-plus 1.0.10
    - Conclusion: don't install weprovide/valet-plus 1.0.9
    - Conclusion: don't install weprovide/valet-plus 1.0.8
    - Conclusion: don't install weprovide/valet-plus 1.0.7
    - Conclusion: don't install weprovide/valet-plus 1.0.6
    - Conclusion: don't install weprovide/valet-plus 1.0.5
    - Conclusion: don't install weprovide/valet-plus 1.0.4
    - Conclusion: don't install symfony/process v3.4.8
    - Conclusion: don't install weprovide/valet-plus 1.0.3
    - Conclusion: remove symfony/process v4.0.8
    - Conclusion: don't install symfony/process v3.4.7
    - Conclusion: don't install weprovide/valet-plus 1.0.2
    - Installation request for symfony/console (locked at v4.0.8) -> satisfiable by symfony/console[v4.0.8].
    - Conclusion: don't install symfony/process v4.0.8
`

**Comment by samgranger** [comment]
Please correct me if I'm wrong, but do you by chance have Laravel's Valet installed? If so, please remove Laravel's version before installing this version.

**Comment by polishedwp** [comment]
I will check but I believe I uninstalled

**Comment by polishedwp** [comment]
Ok, so short story fixed. 

Long story: valet uninstall I had done previously did not fully work. 

I tried: composer global remove laravel/valet but there indications from the console messages that laravel/valet was not in composer.json but laravel/installer installed. 

I dug through the .composer directory but was not able to figure it out. Eventually, I moved everything out of the .composer folder and:
composer global require weprovide/valet-plus 

This actually worked and I am up and running. 

Short story: the uninstall of laravel/valet did not fully uninstall.
